src/control/e2e.spec/select.spec.ts

```diff
@@ -172,3 +172,70 @@ describe(NumberSelectComponent.name, () => {
     element.dispatchEvent(new Event('change'));
   });
 });
+
+@Component({
```

Comment:
the reason for `e2e.spec/select.spec.ts` is that there were some tricky interaction issues with angular itself. This is unrelated to the local form state. It's still a good idea to have an end to end test though, but I'd put it in a separate `local-state.spec.ts` file and maybe use just an `input` instead of the more complex `select`

Reply:
Was not able to trigger a simple change event manually on an input element and have ngrx forms register it properly. Extracted the example into a separate spec file though.